torch/csrc/inductor/aoti_model_runner_cuda.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <torch/csrc/inductor/aoti_model_runner.h>
+
+namespace torch::inductor {
+
+class TORCH_API AOTIModelRunnerCuda : public AOTIModelRunner {
```

Comment:
AOTModelRunnerCuda inherits from AOTIModelRunner, which is guarded by two macros. It may be safer to add the same guard to aoti_model_runner_cuda.[h|cpp]?